File: spring-cloud-config/config-local-server/src/main/java/com/livk/config/ConfigLocalServer.java
package com.livk.config;

import com.livk.commons.spring.SpringLauncher;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.config.server.EnableConfigServer;

/**
 * <p>
 * ConfigLocalServer
 * </p>
 *
 * @author livk
 * @date 2022/7/27
 */
@EnableConfigServer
@SpringBootApplication
public class ConfigLocalServer {
    public static void main(String[] args) {
        SpringLauncher.run(args);
    }
}

File: spring-cloud-config/config-local-server/src/main/kotlin/com/livk/config/ConfigLocalServer.kt
package com.livk.config

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.config.server.EnableConfigServer

/**
 * @author livk
 */
@EnableConfigServer
@SpringBootApplication
open class ConfigLocalServer

fun main(args: Array<String>) {
    runApplication<ConfigLocalServer>(*args)
}
